.cursor/rules/performance.mdc

```diff
@@ -0,0 +1,707 @@
+---
+alwaysApply: false
```

Comment:
1. can you set when to apply this change i.e any change in `application_sdk/*.py`
2. there are some duplicated sections in this file.